lib/index.d.ts

```diff
@@ -477,3 +477,140 @@ export let World: {
+     * chosen to be numerically significant, but visually insignificant.
+     */
+    linearSlop: number;
+    linearSlopSquared: number;
+
+    /**
+     * A small angle used as a collision and constraint tolerance. Usually it is
+     * chosen to be numerically significant, but visually insignificant.
+     */
+    angularSlop: number;
+
+    /**
+     * The radius of the polygon/edge shape skin. This should not be modified.
+     * Making this smaller means polygons will have an insufficient buffer for
+     * continuous collision. Making it larger may create artifacts for vertex
+     * collision.
+     */
+    polygonRadius: number;
+
+    /**
+     * Maximum number of sub-steps per contact in continuous physics simulation.
+     */
+    maxSubSteps: number;
+
+    // Dynamics
+
+    /**
+     * Maximum number of contacts to be handled to solve a TOI impact.
+     */
+    maxTOIContacts: number;
+
+    /**
+     * Maximum iterations to solve a TOI.
+     */
+    maxTOIIterations: number;
+
+    /**
+     * Maximum iterations to find Distance.
+     */
+    maxDistnceIterations: number;
+
+    /**
+     * A velocity threshold for elastic collisions. Any collision with a relative
+     * linear velocity below this threshold will be treated as inelastic.
+     */
+    velocityThreshold: number;
+
+    /**
+     * The maximum linear position correction used when solving constraints. This
+     * helps to prevent overshoot.
+     */
+    maxLinearCorrection: number;
+
+    /**
+     * The maximum angular position correction used when solving constraints. This
+     * helps to prevent overshoot.
+     */
+    maxAngularCorrection: number;
+
+    /**
+     * The maximum linear velocity of a body. This limit is very large and is used
+     * to prevent numerical problems. You shouldn't need to adjust this.
+     */
+    maxTranslation: number;
+    maxTranslationSquared: number;
+
+    /**
+     * The maximum angular velocity of a body. This limit is very large and is used
+     * to prevent numerical problems. You shouldn't need to adjust this.
+     */
+    maxRotation: number;
+    maxRotationSquared: number;
+
+    /**
+     * This scale factor controls how fast overlap is resolved. Ideally this would
+     * be 1 so that overlap is removed in one time step. However using values close
+     * to 1 often lead to overshoot.
+     */
+    baumgarte: number;
+    toiBaugarte: number;
+
+    // Sleep
+
+    /**
+     * The time that a body must be still before it will go to sleep.
+     */
+    timeToSleep: number;
+
+    /**
+     * A body cannot sleep if its linear velocity is above this tolerance.
+     */
+    linearSleepTolerance: number;
+    linearSleepToleranceSqr: number;
+
+    /**
+     * A body cannot sleep if its angular velocity is above this tolerance.
+     */
+    angularSleepTolerance: number;
+    angularSleepToleranceSqr: number;
+}
+
+export let internal: {
+    Settings: Settings;
```

Comment:
Out of curiosity: Are the other internal objects also supposed to be exposed or are they supposed to be private? If they should be exposed in the future, you could maybe add them as `any` already.

Reply:
They are not private, just not supposed to be used externally for now, unless there is a specific case (see #95 for example).

Reply:
Also it is useful for myself, when working on the library! :)